### DISCUSSION TITLE
Auto-merge the PRs where the only thing that has changed is the commit SHA and a version increment

### DISCUSSION THREAD

**Discussion by antkmsft** [microsoft/vcpkg#32431] [Q&A]
### Is your feature request related to a problem? Please describe.

It is not a problem, but I think port updates can be faster if they were merged faster, and some of them probably can be merged faster - I am talking the situation when a port update comes from a known GitHub contributor who has updated the port before, and the PR is only updating the commit SHA and increments the package version.

Look at NPM, for example - after a change is pushed to a registry, it is available within minutes. Vcpkg has manual review process, and I understand that, but I think that there are some cases that can be safely automated.

### Proposed solution

Create a list of "known contributors" for each port. If an automation sees a PR from that person, and the PR only updates the commit SHA and increments the version, auto-approve and auto-merge it.

In addition, there could be a requirement that the PR title would be in the format of "[port-name] *", where port-name may contain wildcards, if the PR is updating multiple packages - just so that commit history looks good.

**Comment by MonicaLiu0311** [comment]
Go to the discussion board to communicate.
